src/examples/core/fetch-asset/curl.sh

```
@@ -0,0 +1,12 @@
+# Fetch an NFT using the DAS API
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)